### DISCUSSION TITLE
Cannot connect in bridge mode, but OK as simulator

### DISCUSSION THREAD

**Discussion by hpcolus** [zepp-health/zeppos-forum#196] [Developer Feedback]
The simulator runs OK with no problems but when I try the bridge mode, it does not.  I activated in the phone the bridge mode in Zepp app and it says is OK, but when running the ZEUS CLI command connect, It complains there is no connection.  
![image](https://github.com/zepp-health/zeppos-forum/assets/104551107/22293707-92e5-4f36-b0b4-a4a895026605)

This is the screenshot of the login connection status
![image](https://github.com/zepp-health/zeppos-forum/assets/104551107/a0eed105-16b7-4d18-b62f-96ed8d6ffd6a)

When I click on bridge icon, I am presented always with the login page.  I put the credentials but nothing happens. Is there anything I should do or is this a bug?
![image](https://github.com/zepp-health/zeppos-forum/assets/104551107/c7d67c68-b7d9-4403-997c-c609e44d2ce2)

**Comment by MyoungXUE** [comment]
Hello, I suggest that you can reinstall the simulator, open the simulator and then click on the top left corner of the avatar to log in, and then use the bridge mode.

**Reply by hpcolus** [reply]
It did not work.
I uninstalled and installed new version 1.2.1 on my desktop computer.  I also installed in a new laptop and it is the same.  Nothing,  Bridge cannot be activated in the simulator.   All I get is the login page, but the bridge is not activated

**Comment by DotIN13** [comment]
I had the same problem, and after I logged into my developer account over at https://developer.zepp.com/os/develop, uploaded an avatar, passed the developer verification, and created a test watchface project, I was able to login to the simulator without an issue.
